docs/crictl.md

```diff
@@ -15,6 +17,15 @@ sudo tar zxvf crictl-$VERSION-linux-amd64.tar.gz -C /usr/local/bin
 rm -f crictl-$VERSION-linux-amd64.tar.gz
 ```
 
+- using `curl`:
+
+```sh
+VERSION="v1.11.1"
```

Comment:
could we use the same version v1.13.0 as above

Reply:
Thanks for the comment, I have made the change. 
I will resolve this convo.